sphinx/util/inventory.py

```diff
@@ -22,7 +20,7 @@
     from sphinx.builders import Builder  # NOQA
     from sphinx.environment import BuildEnvironment  # NOQA
 
-    Inventory = Dict[text_type, Dict[text_type, Tuple[text_type, text_type, text_type, text_type]]]  # NOQA
+    Inventory = Dict[str, Dict[str, Tuple[str, str, str, str]]]  # NOQA
```

Comment:
It would be better to move this definition to `sphinx.util.typing` module. This is commonly used in two or more modules.

Reply:
Done in #5817